auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -191,13 +196,16 @@ import i18n from '@/plugins/i18n'
   }
 })
 export default class EntityManagement extends Mixins(AccountChangeMixin, NextPageMixin) {
+  readonly CommonUtils = CommonUtils
+
   @Prop({ default: '' }) private orgId: string;
   private removeBusinessPayload = null
   private dialogTitle = ''
   private dialogText = ''
   private messageTextList = i18n.messages[i18n.locale]
   private isLoading = true
   private resetPasscodeEmail: string = null
+  businessIdentifier: string = null
```

Comment:
null or empty string?

Reply:
It doesn't matter Both null and empty string are falsy, and I made sure that `CommonUtils.isCooperativeNumber()` handles it correctly (and I added a unit test for it as well).